Homework3/Task4.py

```diff
@@ -0,0 +1,17 @@
+# Напишите программу, которая будет преобразовывать десятичное число в двоичное.
+
+#  Пример:
+
+# 45 -> 101101
+# 3 -> 11
+# 2 -> 10
+
+a = int(input('Введите число: '))
+lst = []
+n = a
+while(n >= 1):
+    res = n % 2
+    n /= 2
```

Comment:
Ваша программа не будет работать если не использовать целочисленное деление
```suggestion
    n //= 2
```
Поэтом вопрос: вы вообще запускали код?

Reply:
Даже обидно. Конечно я запускаю код. В голове без проверки я не умею код писать от слова совсем. У меня все работает. Могу скрины прислать в личку ГБ или телеги, что все работает

Reply:
Но вот просто закономерный, я у себя проверял и у меня происходила ошибка. Впрочем, я тоже могу ошибаться.

Reply:
Верю, что была ошибка. на 2м семинаре в работе в группах мы с вами запускали 1 и тот же код. у вас работал, у меня нет. теперь обратная история. Я скинула в ГБ скрин, что работает у меня. А себе возьму исправление ваше на заметку. СПАСИБО!